CMakeLists.txt

```diff
@@ -999,6 +999,9 @@ else()
 
   if(APPLE)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fvisibility=default -DGTEST_HAS_TR1_TUPLE=0")
+    if(ARM)
+      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-aligned-allocation")
```

Comment:
I missed this earlier, why this change? Should it be applied to `CMAKE_C_FLAGS` too?

Reply:
This is a C++-17 specific flag for clang.

Reply:
We set the macOS deployment target to 10.13 which doesn't support aligned alloctation. I think support for it was added in 10.14 so unless we bump the deployment target this has to be added. To fix compilation I only had to add it to CMAKE_CXX_FLAGS.

Reply:
CMAKE_C_FLAGS don't care because it's a C++-17 flag :)

Reply:
I'm out of the loop with everything C++, I didn't realize there was a new function for aligned allocation.